continuum/scenarios/permutations.py

```diff
@@ -0,0 +1,72 @@
+from typing import Callable, List
+
+import torch
+
+from continuum.datasets import _ContinuumDataset
+from continuum.scenarios import TransformationIncremental
+
+from torchvision import transforms
+
+
+class Permutations(TransformationIncremental):
+    """Continual Loader, generating datasets for the consecutive tasks.
+
+    Scenario: Permutations scenarios, use same data for all task but with pixels permuted.
+    Each task get a specific permutation, such as all tasks are different.
+
+    :param cl_dataset: A continual dataset.
+    :param nb_tasks: number of tasks in the continuum.
+    :param base_transformations: List of transformation to apply to all tasks.
+    :param seed: initialization seed for the permutations.
+    """
+
+    def __init__(
+            self,
+            cl_dataset: _ContinuumDataset,
+            nb_tasks: int,
+            base_transformations: List[Callable] = None,
+            seed=0
+    ):
+        list_transformations = []
+        g_cpu = torch.Generator()
+        list_seed = torch.randperm(1000, generator=g_cpu)[:nb_tasks]
```

Comment:
This code is not reproducible, because the `list_seed` is different every time. This variable should be given by the user.

Reply:
I think, it is reproducible now
https://github.com/Continvvm/continuum/blob/master/continuum/scenarios/permutations.py#L32-L34